github_run.py

```diff
@@ -92,6 +93,7 @@ def get_files(self):
     gs = galaxy_social(args.preview, args.json_out)
     try:
         message = gs.process_files(files_to_process)
+        github.comment(message)
     except Exception as e:
-        message = e
-    github.comment(message)
+        github.comment(traceback.format_exc())
```

Comment:
Are we sure we want to put tracebacks as github comments? I guess printing this as part of the stderr on CI is enough?

Reply:
So would it be to raise the error and comment `e` to GitHub?

Reply:
Is it meaningful to comment for example `Expecting value: line 2 column 1 (char 1)` to github?

Reply:
I don't think its useful to put error message of any kind to the user. Maybe something like "Something went wrong, an Admin will take a look." But that all. A user can not deal with the errors either way and an Admin should be able to look at the CI looks to see all the nifty details.

Reply:
But for example, the error that I mentioned above seems to be caused by the post md file and the metadata format is wrong.
So, should we comment with the fixed text for all types of errors?

Reply:
And for example this error:
```
Invalid metadata in posts/2024-05-14_job_opening_and_matrix_test.md.
while scanning for the next token
found character '@' that cannot start any token
in "", line 6, column 7:
- @anikaerxleben:matrix.org
^
```
is indicated a problem in the post md file.
Shouldn't this error be visible for the user to fix the post file? or do just the admins fixing the errors?

Reply:
This error is probably not meaningful to our users. Either we provide a better error message, that is understandable or our linter needs to catch this before.

Reply:
OK, what text should the linter provide as a comment on Github?

Reply:
Something meaningful to an end user.
Please check your links. It seems your gihub handle is not valid ... I don't know.

Its also not super important for today, can be improved over time. @wm75 decision :)

Reply:
Yeah, right now this is not very important, and lets not start to improve the linting as part of this.
For me, it's totally fine to go into the CI ressults for now.
Once we have something satisfying enough that we want to promote it, we can start improving error messages.